### PR TITLE
fix redirect

### PR DESCRIPTION
**Pull request by 3442853561**
fix issue https://github.com/rust-lang/rust-www/issues/742 and https://github.com/rust-lang/rust-www/issues/725
fix some potential problems

### CODE REVIEW

File: _layouts/redirect.html [hunk 1]
```diff
@@ -9,10 +9,12 @@
       var languages = {
         "en": "en-US",
         "en-US": "en-US",
-        "fr": "fr",
-        "fr-FR": "fr", // Prevents 404 on existing links
+		"es": "es-ES",
+		"es-ES":"es-ES",
```
Comment (steveklabnik) [inline _layouts/redirect.html]: it looks like some of this is using tabs, can you make it all spaces please?

File: _layouts/redirect.html [hunk 2]
```diff
@@ -9,10 +9,12 @@
...
+        "fr": "fr-FR",
```
Comment (steveklabnik) [inline _layouts/redirect.html]: 👍 ❤️

File: _layouts/redirect.html (hunk 1 above)
Comment (3442853561) [inline _layouts/redirect.html]: okay, I will fix it

**Comment by steveklabnik** [comment]
Thank you!

**Pull request merged**